packages/www/components/Dashboard/ErrorRecordDialog/index.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Button,
+  Flex,
+  AlertDialog,
+  AlertDialogTitle,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogCancel,
+  Heading,
+  Text,
+} from "@livepeer.com/design-system";
+
+const ErrorRecordDialog = ({
+  isOpen,
+  onOpenChange,
+}: {
+  isOpen: boolean;
+  onOpenChange: (isOpen: boolean) => void;
+}) => {
+  return (
+    <AlertDialog open={isOpen} onOpenChange={onOpenChange}>
+      <AlertDialogContent css={{ maxWidth: 450, px: "$5", pt: "$4", pb: "$4" }}>
+        <AlertDialogTitle as={Heading} size="1">
+          Error
+        </AlertDialogTitle>
+        <AlertDialogDescription
+          as={Text}
+          size="3"
+          variant="gray"
+          css={{ mt: "$2", lineHeight: "22px" }}>
+          You cannot change recording preferences while a session is active
```

Comment:
This text needs to be parametrized as well, since we are showing the same thing for when trying to configured multistream and recording.

Can just change to the same thing but replacing `recording` with `multistream` like
```
You cannot change multistream preferences while a session is active
```

Then I guess the component can just be renamed to `ErrorDialog` since it will just receive a generic text to show as the error.